### ISSUE TITLE
04.00 erroneous/inconsistent labels

### ISSUE THREAD

**Issue by alexklibisz** [CodeNeuro/neurofinder#25]
Hi all, 

It looks like the labels on 04.00 might be incorrect in some places. I'm not an expert with this kind of data (more CS background), but there seems to be many instances of non-neurons that look similar to neurons from other datasets but are not marked in this dataset. I've marked some examples from the `thumbnail.png` file below. Moreover, the submissions for 04.00.test seem to have a very low precision even among the top solutions. Maybe it's possible the heldout dataset has similar problems? It seems that @mjlm is the original contributor for this dataset. I appreciate any feedback that can be offered.

Here is a video of the 04.00 dataset for easy reference: https://youtu.be/z-IMtnw8gfs
and the 04.01 dataset: https://youtu.be/eg1Gs1a4aUg

![image](https://user-images.githubusercontent.com/8015228/27055966-32803d52-4f94-11e7-8208-3ca12207646f.png)

**Comment by mjlm** [comment]
Hi Alex,

Yes, I recorded and annotated this dataset.

You've identified a few important challenges when working with this kind of data.

Firstly, some neurons might appear bright in the mean image but never become active, i.e. their brightness timeseries is just noise. This is why it is generally not sufficient to look at the mean image to segment the neurons, but necessary to include temporal information. This is one possible reason why I didn't label the neurons that you marked with arrows.

Secondly, when we annotate such datasets by hand for our own experiments, we might exclude neurons that look "unhealthy" (based on subjective experience). 

So these labels are not incorrect, they are simply optimized for a real experiment rather than for providing accurate scores during algorithm development. I also remember that this dataset had quite a low signal to noise ratio and we included it as a challenging example.

Regarding low precision in general, I think automated algorithms for cell selection should never be as selective as human annotation. It's always easier/faster for the experimenter to remove some false positives than to go back to the raw data to check if the algorithm missed something.
